src/github.com/getlantern/flashlight/pubsub/pubsub.go

```diff
@@ -0,0 +1,24 @@
+// package pubsub provides a global publish and subscribe infrastructure
+// for lantern
+package pubsub
+
+import (
+	"reflect"
+
+	"github.com/asaskevich/EventBus"
+)
+
+var (
+	bus = EventBus.New()
+)
+
+// Pub publishes the given interface to any listeners for that interface.
+func Pub(topic interface{}) {
```

Comment:
OK, I just added some changes here, but actually to use strings instead of ints. I want to encourage compile time checking here, which is the whole purpose of doing it by type, but I think the int approach is a happy medium of sorts (i.e. it would feel very odd to throw in arbitrary ints as topics, encouraging callers to take a peek at how it works and to add topics to pubsub.go).